pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1255,6 +1271,9 @@ func doInactivate(ctx *domainContext, status *types.DomainStatus, impatient bool
 	if err == nil && domainID != status.DomainId {
 		status.DomainId = domainID
 	}
+	if status.IsContainer {
+		impatient = true
```

Comment:
A comment about why we wait only for 1/10th of max delay would be useful

Reply:
Magic - perhaps black magic ;-)
I'll add a comment about the container being read-only hence we can tear it down more agressively.